jacoco-maven-plugin/src/org/jacoco/maven/ReportAggregateMojo.java

```diff
@@ -91,6 +92,14 @@ public class ReportAggregateMojo extends AbstractReportMojo {
 	@Parameter(defaultValue = "false")
 	private boolean includeCurrentProject;
 
+	/**
+	 * Include all projects from the reactor.
+	 *
+	 * @since 0.8.13
+	 */
+	@Parameter(defaultValue = "false", property = "jacoco.onlyReactorProjects")
+	private boolean onlyReactorProjects;
```

Comment:
Why is this property called `only...` and not `all...`?

From the javadoc seems more appropriate to call it `allReactorProjects`, or perhaps `allModules` or `allSubModules`?

Reply:
I was not really sure because the code currently reads only all dependencies declared in the coverage/aggregator project (dependencies that are part of the reactor). 
If the flag is activated, the code will not use any current protect dependencies but only(All :))ReactorProjects. 
So yes, something like `allReactorProjects` would also work, except it doesn't tell the user the configuration no longer reads the project's dependencies.
Doh, this looks complicated to explain so that it might be wrong :).

no real strong opinion. @Godin what do you think?